fix: Fix issues in manifests in apps repo

mimotej opened the pull request:
This PR tries to fix various issues in our manifests. Changes were made, when CI which will be introduced in PR https://github.com/operate-first/apps/pull/1318, failed with error on some resource. 
Part of: https://github.com/operate-first/SRE/issues/107

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **tumido** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @tumido` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Fapps).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/apps/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["tumido"]} -->

HumairAK commented:
/lgtm

Let's also get @larsks  to take a look (in particular the NNCP fixes)

HumairAK commented:
Side note: I don't think we need 7 commits for this, one commit stating "Update resources to fit openapi specs" or something is fine I think.

larsks commented on cluster-scope/overlays/prod/moc/zero/nodenetworkconfigurationpolicies/moc-nfs-network.yaml:
```diff
@@ -3,7 +3,6 @@ kind: NodeNetworkConfigurationPolicy
 metadata:
   name: moc-nfs-network-enp4s0f1np1
 spec:
-  parallel: true
```
We were setting `parallel: true` explicitly to work around [a problem][1] (see a37d254a2d). I suspect it's safe to remove, although I'd like to follow up with someone on the networking side to see if this is actually a bug in the openapi spec.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1967771

mimotej replied:
Tried tests again just to be sure and it really gives an error (on picture below) which might suggest a bug in the openapi spec... Anyway these changes are done so that we don't have to delete whole zero (because rest of manifest seem fine), so it shouldn't matter too much anyway.
![image](https://user-images.githubusercontent.com/47832967/144819033-b819067f-ef27-442e-941c-a47519aaf73e.png)

sesheta commented:
New changes are detected. LGTM label has been removed.

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **humairak** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Fapps).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/apps/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["humairak"]} -->

mimotej commented:
> Side note: I don't think we need 7 commits for this, one commit stating "Update resources to fit openapi specs" or something is fine I think.

Good point :+1: . Squashed and rebased commits. Which showed me that there were some changes in schemas, so I will create a PR with new schemas to be add to schema-store.

mimotej commented:
/retest

sesheta commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/operate-first/apps/pull/1400#pullrequestreview-826742142" title="Approved">larsks</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Fapps).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/operate-first/apps/blob/master/OWNERS)~~ [larsks]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

HumairAK commented:
/lgtm
